### PR TITLE
Royal Court Crashing Merge

### PR DESCRIPTION
**Pull request by GingerAvalanche**
Just a consolidation of files before we try to continue fixing Royal Court crashing everything

Be sure to look over all the changes and make sure I didn't delete anything you need

### CODE REVIEW

File: common/buildings/ancient_magic_city_buildings.txt [hunk 1]
```diff
@@ -1292,10 +1292,7 @@ guild_halls_01 = {
 		modifier = {
 			add = 500
 			scope:holder = {
-				OR = {
-					has_government = magocracy_government
-					has_government = republic_government
-				}
+				has_government = republic_government
```
Comment (Donekulda) [inline common/buildings/ancient_magic_city_buildings.txt:1295]: has_government = magocracy_government

Comment (Donekulda) [inline common/buildings/ancient_magic_city_buildings.txt:1295]: has_government = magocracy_government is missing in this OR and many other ORs in this file

File: common/deathreasons/01_magic_deaths.txt [hunk 2]
```diff
@@ -19,9 +19,4 @@ death_mana_devouring_2 = {
 	}
 	priority = 70
 	icon = "death_disease.dds"
```
Comment (Donekulda) [inline common/deathreasons/01_magic_deaths.txt:21]: death because of soul forging

File: common/decisions/01_magic_decisions.txt [hunk 3]
```diff
@@ -88,50 +307,3 @@ reset_mana_system_decision = {
 
 	ai_will_do = { base = 0 }
 }
```
Comment (Donekulda) [inline common/decisions/01_magic_decisions.txt:309]: soul forging decision

File: common/game_concepts/ancient_magic_concepts.txt [hunk 4]
```diff
@@ -67,12 +68,24 @@ school_of_magic = {
...
+
 undead = {
-    alias = { greater_undead }
+	texture = "gfx/interface/icons/regimenttypes/lesser_undead.dds"
+    alias = { lesser_undead greater_undead }
 }
 
 magocracy = {
-    alias = { magocracies magocratic magocracy_government }
+    alias = { magocracies magocratic magocratic_government }
```
Comment (Donekulda) [inline common/game_concepts/ancient_magic_concepts.txt:88]: if changed then atleast alias = { magocracies magocratic magocratic_government magocracy_government }

File: localization/english/ancient_magic_custom_description_l_english.yml [hunk 5]
```diff
@@ -1,5 +0,0 @@
-﻿l_english:
```
Comment (Donekulda) [inline localization/english/ancient_magic_custom_description_l_english.yml:1]: why was this deleted ? The reason there is this loc, is becaus evanilla doesnt has or, didnt have it before, dont know about now

File: localization/english/ancient_magic_education_l_english.yml [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
+ trait_education_arcane_1_desc: "[ROOT.Char.GetFirstNameNoTooltip] attended [ROOT.Char.GetHerHis] lessons, though that's about all that can be said. [ROOT.Char.GetHerHis|U] attempts to recall their studies are laughable, often surpassed by even the most inexperienced of apprentices."
+ trait_education_arcane_2: "Magic Journeyman"
+ trait_education_arcane_2_desc: "[ROOT.Char.GetFirstNameNoTooltip] is a mediocre arcane historian. Though [ROOT.Char.GetSheHe] attended lessons on magical history and theory, [ROOT.Char.GetSheHe] may have gotten less out of them than other students."
+ trait_education_arcane_3: "Magic Expert"
+ trait_education_arcane_3_desc: "[ROOT.Char.GetFirstNameNoTooltip] is a well respected expert on magical theory, capable of holding their own against most magical historians... if society permits."
+ trait_education_arcane_4: "Magical Theorist"
+ trait_education_arcane_4_desc: "[ROOT.Char.GetFirstNameNoTooltip] is one of the foremost experts on magic in the world. [ROOT.Char.GetSheHe|U] is a prodigy in the theory of spell casting, and could become one of the greatest practitioners."
+
  #education
- education_arcane:0 "Arcane Education"
```
Comment (Donekulda) [inline localization/english/ancient_magic_education_l_english.yml:13]: why were the 0 deleted ?

File: common/traits/01_ancient_magic_traits.txt [hunk 7]
```diff
@@ -227,39 +231,6 @@ mana_devouring_2 = {
 	level = 2
 }
 
-mana_burn = {
```
[the remark this answers is not in the document]
Comment (Donekulda) [reply common/traits/01_ancient_magic_traits.txt:230]: trait that is prepared for future use, it will be caused if one uses spell over ones body capacity, in another sense if you cast spell which u cant cast without equipment or bilding it will cause mana burn

File: common/religion/doctrines/00_core_tenets.txt [hunk 8]
```diff
@@ -0,0 +1,2705 @@
+﻿doctrine_core_tenets = {
```
[the remark this answers is not in the document]
Comment (Donekulda) [reply common/religion/doctrines/00_core_tenets.txt:1]: i deleted it so idk

File: common/modifiers/01_ancient_magic_modifiers.txt [hunk 9]
```diff
@@ -1,130 +1,99 @@
...
-	icon = health_positive
-	health = 1.5
-}
-
-biomantic_fertility_modifier = {
-	icon = health_positive
-	fertility = 10
-}
-
-biomantic_infertility_modifier = {
```
[the remark this answers is not in the document]
Comment (Donekulda) [reply common/modifiers/01_ancient_magic_modifiers.txt:34]: already know

File: common/on_action/01_ancient_magic_on_actions.txt [hunk 10]
```diff
@@ -55,28 +55,13 @@ death_stare_action = {
 random_yearly_everyone_pulse = {
 	on_actions = {
 		check_lifetime_flags_for_immortal
-		update_immortal_years_since_master_of_time_spell
-	}
-}
-
-update_immortal_years_since_master_of_time_spell= {
```
[the remark this answers is not in the document]
Comment (Donekulda) [reply common/on_action/01_ancient_magic_on_actions.txt:62]: idk im just asking

File: localization/english/ancient_magic_education_l_english.yml (hunk 6 above)
Comment (Donekulda) [reply localization/english/ancient_magic_education_l_english.yml:13]: again i really dont know, im just asking

File: common/game_concepts/ancient_magic_concepts.txt (hunk 4 above)
Comment (GingerAvalanche) [reply common/game_concepts/ancient_magic_concepts.txt:88]: https://github.com/absylim/Ancient-Magic-CK3-Mod/blob/850f31ec7f63c69ac68505f59ddf74c7d7965b72/localization/english/ancient_magic_government_l_english.yml#L5

Note that this link is into your repo.

`magocracy_government` doesn't exist. `magocratic_government` does. That's why I changed it. I changed it to fix an error, I'm not going to add the error back in.

File: common/game_concepts/ancient_magic_concepts.txt [hunk 11]
```diff
@@ -181,16 +199,3 @@ enchanting_aura_spell = {}
 sculpt_mind_spell = {}
 
 master_of_puppets_spell = {}
-
-## Men At Arms
```
[the remark this answers is not in the document]
Comment (GingerAvalanche) [reply common/game_concepts/ancient_magic_concepts.txt:185]: No. They were moved to right above `magocracy`. I combined `lesser_undead` with `undead` to fix a bug. Could've moved `undead` down here, but I decided to put concepts that fit in vanilla systems higher in the file. (Men at arms exist in vanilla, but spells don't)

File: common/scripted_guis/magic_scripted_gui.txt [hunk 12]
```diff
@@ -145,7 +142,7 @@ increase_times_repeated = {
 		AND = {
 			OR = {
 				NOT = { is_magic_aura_trigger = { SPELL = scope:spell } }
-				scope:potential_repeat_spell < mage_level
+				scope:potential_repeat_spell <= mage_level
```
[the remark this answers is not in the document]
Comment (GingerAvalanche) [reply common/scripted_guis/magic_scripted_gui.txt:145]: Character variables are accessed with `var`, not script_values. `mage_value` is a script_value, not a character variable.

File: common/scripted_triggers/01_ancient_magic_secret_type_trigger.txt [hunk 13]
```diff
@@ -0,0 +1,25 @@
+﻿secret_mage_is_valid_trigger = {
+	$OWNER$ = {
+		NOT = {
+			has_trait = witch
```
[the remark this answers is not in the document]
Comment (GingerAvalanche) [reply common/scripted_triggers/01_ancient_magic_secret_type_trigger.txt:4]: Good catch. Thought I'd changed it to the mage trigger

File: localization/english/ancient_magic_education_l_english.yml (hunk 6 above)
Comment (GingerAvalanche) [reply localization/english/ancient_magic_education_l_english.yml:13]: They're only used by Paradox dev tools for debugging purposes. For us, they do nothing but mess with the coloring in our IDEs. I removed them to make it color the keys differently so it would be easy to tell what text is keys and what text is values.

File: localization/english/gui/ancient_magic_title_view_l_english.yml [hunk 14]
```diff
@@ -1,3 +0,0 @@
-﻿l_english:
```
[the remark this answers is not in the document]
Comment (GingerAvalanche) [reply localization/english/gui/ancient_magic_title_view_l_english.yml:1]: Because the button for granting to a mage was removed, as was the effect. Grant To Mage was only there as a workaround to make sure a magocratic character was appointed to lead a city of magic, but it never actually worked.

File: localization/english/ancient_magic_education_l_english.yml (hunk 6 above)
Comment (Donekulda) [reply localization/english/ancient_magic_education_l_english.yml:13]: ok

**Review by Donekulda — APPROVED** [review]
